index.js

```diff
@@ -0,0 +1,28 @@
+const express = require("express");
+const app = express();
+const PORT = process.env.PORT || 4000;
+const homepages = require("./routers/homepages");
+const auth = require("./routers/auth");
+const mypage = require("./routers/mypage");
+const websites = require("./routers/websites");
+const ideas = require("./routers/ideas");
+const skills = require("./routers/skills");
+
+const bodyParserMiddleWare = express.json();
+app.use(bodyParserMiddleWare);
+
+const corsMiddleWare = require("cors");
+app.use(corsMiddleWare());
```

Comment:
cors should be the 1st middleware! Also, i know the template is not like this but i prefer all my imports at the top 😄